### DISCUSSION TITLE
Autofix proposal for `CPY`

### DISCUSSION THREAD

**Discussion by durumu** [astral-sh/ruff#6516] [Ideas]
I would find it very useful if a `flake8-copyright` rule could auto-add a hardcoded disclaimer, e.g. `# Property of ExampleCo`. I'm happy to implement this autofix myself. However, I'm not sure how this would interact with the existing `notice-rgx`. Here are a few ideas, though I'm very open to others:

- Make a new rule, e.g. `CPY002`, that uses a hardcoded string rather than a regex, and add this string to the top of the file whenever it is not present there.
- Have some separate hardcoded string, e.g. `notice-raw = "Property of ExampleCo"`, that is used for the autofix. In this case, the rule would be if `notice-rgx` doesn't match `notice-raw`, `notice-raw` would be added to the top of the file. 
- Make autofix available only if the `notice-rgx` is hardcoded.

Would love to hear your thoughts!

**Comment by zanieb** [comment]
Hey @durumu

Mind chiming in on the existing issue at https://github.com/astral-sh/ruff/issues/5709 instead?

**Reply by durumu** [reply]
Oh no, I missed that! Thank you.
